Makefile

```diff
@@ -14,3 +14,5 @@ test: compile
 typecheck:
 	$(REBAR) dialyzer
 
+ci:compile
+	$(REBAR) dialyzer && $(REBAR) as test do eunit
```

Comment:
FYI: every line in a Makefile recipe is implicit `&&`d with the line before. It's conventional to put each execution on its own line.

```
$(REBAR) dialyzer
$(REBAR) as test do eunit
```